debile/utils/xmlrpc.py

```diff
@@ -141,3 +142,13 @@ def get_proxy(config, auth_method):
                 ca_certs=xml.get('ca_certs', "/etc/ssl/certs/ca-certificates.crt")
             ), allow_none=True)
     return proxy
+
+
+def get_auth_method(args, config):
+    auth_method = args.auth_method
+
+    if 'xmlrpc' in config:
+        if 'auth_method' in config['xmlrpc']:
+            auth_method = config['xmlrpc']['auth_method']
```

Comment:
This will raise an exception if there is no 'xmlrpc' in the config file, please catch it (and add a unit test for that case).